Docs/user-guides/reference/02-working-with-openl-tables/03-table-types/02-decision-table/13-rules-tables.md

```
@@ -0,0 +1,41 @@
+##### Rules Tables
+
+A **rules table** is a regular decision table with vertical and optional horizontal conditions where the structure of the condition and return columns is explicitly declared by a user by starting column headers with the characters specific for each column as described in [Decision Table Structure](decision-table-overview-and-structure.md#decision-table-structure).
+
+By default, each row of the decision table is a separate rule. Even if some cells of condition columns are merged, OpenL Tablets treats them as unmerged. This is the most common scenario.
+
+Vertical conditions are marked with the Cn and MC1 characters. The MC1 column plays the role of the Rule column in a table. It determines the height of the result value list. An example is as follows.
+
+![](../../../../ref_guide_images/b912a0b5ee38ce8aae0ffa6fc92a3234.png)
+
+*A Decision table with merged condition values*
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)